lib/matplotlib/dates.py

```diff
@@ -83,10 +83,11 @@
      Out[1]: 732401
 
 All the Matplotlib date converters, tickers and formatters are timezone aware.
-If no explicit timezone is provided, :rc:`timezone` is assumed.  If you want to
-use a custom time zone, pass a `datetime.tzinfo` instance with the tz keyword
-argument to `num2date`, `.Axis.axis_date`, and any custom date tickers or
-locators you create.
+If no explicit timezone is provided, :rc:`timezone` is assumed, provided as a
+string.  If you want to use a different timezone, pass the *tz* keyword
+argument of `num2date` to any date tickers or locators you create.  This can
```

Comment:
This sentence no longer says the same thing. Previously it said to pass `tz` to `num2date` or the other options. Now it says to pass `num2date` to the other options.

Reply:
I do not read it like that. To me it sounds more like "use the same tz argument that you passed to num2date". But I agree that it probably can be made clearer (obviously, as you read it differently).

(Btw, was suggested here: https://github.com/matplotlib/matplotlib/pull/22196#discussion_r785758344 )